src/layouts/CasestudiesLayout.astro

```diff
@@ -0,0 +1,61 @@
+---
+import BaseLayout from '../layouts/BaseLayout.astro';
+import AboutusNavigation from '../components/AboutusNavigation.astro';
+import { Image } from "astro:assets";
+import { getCollection } from "astro:content";
+import Header from '../components/Header.astro';
+import onlineImage from '../assets/bioimage-archive/online_tutorial_image.png';
+import paperImage from '../assets/bioimage-archive/BioImage_Archive_paper.png';
+const allPosts = await getCollection("case_studies");
+---
+<html lang="en" class="vf-js">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" type="image/svg+xml" href="/bioimage-archive/favicon.svg" />
+    <meta name="viewport" content="width=device-width" />
+    <meta name="generator" content={Astro.generator} />
+
+    <!-- CSS only -->
+    <link rel="stylesheet" href="https://assets.emblstatic.net/vf/v2.5.12/css/styles.css">
+    <!-- JS -->
+    <script src="https://assets.emblstatic.net/vf/v2.5.12/scripts/scripts.js"></script>
+  <title>Case Studies</title>
+</head>
+<body class="vf-body">
+  <header id="masthead-black-bar" class="clearfix masthead-black-bar | ebi-header-footer vf-content vf-u-fullbleed"></header>
+  <script defer="defer" src="//ebi.emblstatic.net/web_guidelines/EBI-Framework/v1.4/js/script.js"></script>
+  <link rel="stylesheet" href="https://assets.emblstatic.net/vf/v2.5.8/assets/ebi-header-footer/ebi-header-footer.css" type="text/css" media="all">
+ 
+  <Header />
+  <hr>
+  <section class="embl-grid embl-grid--has-centered-content">
+    <div>
+      <AboutusNavigation />
+    </div>
+    <div class="vf-content">
+      <slot />
+    </div>
+    <div>
+          <h3>Further information</h3>
```

Comment:
This section gets re-used across all the about us pages.

1) would it be better to make this a component you import to each page for re-use
2) i'm personally not a fan of the design: it currently occupies a location that my brain is programmed to ignore because that's where adverts go, and is generally adding noise the the main section of the page. I feel like having it at the bottom of the main page content would be a more natural spot if we actually want to direct people to those pages. But i won't block this pr on this design discussion.

Reply:
Did the component; happy to discuss about design.